src/components/AddItem/TagInput/TagInput.jsx

```diff
@@ -0,0 +1,55 @@
+import { RemoveIcon } from '@/components/common/Buttons';
+import formStyles from '@/styles/helpers/formHelpers.module.scss';
+import styles from './TagInput.module.scss';
+
+const TagInput = ({ tagInput, setTagInput, tags, handleInputChange }) => {
```

Comment:
💊 제안
props명과 컴포넌트 명들이 중복되는 정보들을 가지고 있어요. 
아래처럼 작성하셔서 좀 더 명확하게 네이밍하시는 것을 추천드려요!

```suggestion
const TagInput = ({ tags, handleInputChange }) => {
```

또한 setTagInput와 handleInputChange를 둘 다 받고 있습니다.
지금 제가 파악하기로는 tagInput과 같은 인풋값이 외부에서 필요한 것이 아니라서 컴포넌트 내부에서 처리하는 것이 좋아 보입니다.